app/webpacker/controllers/tom_select_controller.js

```diff
@@ -24,4 +29,10 @@ export default class extends Controller {
   disconnect() {
     if (this.control) this.control.destroy();
   }
+
+  #placeholder() {
+    const optionsArray = Array.apply(null, this.element.options);
```

Comment:
I think I prefer using the spread operator: 
```suggestion
    const optionsArray = [...this.element.options];
```

What do you think?

Reply:
:+1: that's better